src/server/procedures/myForestV2/leaderboard.ts

```diff
@@ -92,6 +96,7 @@ export const leaderboardProcedure = procedure
       name: gift.giverName,
       units: gift.quantity,
       unitType: 'tree',
+      purpose: gift.purpose,
     }));
 
     // Fetch the top 10 gifters for the profile(s)
```

Comment:
Won't we need to add `purpose` for `most trees` array elements as well?

Reply:
yes, missed that.